packages/firestore/src/core/firestore_client.ts

```diff
@@ -331,6 +321,16 @@ export class FirestoreClient {
             user
           )
         : new MemorySharedClientState();
+      const persistence: IndexedDbPersistence = new IndexedDbPersistence(
```

Comment:
It looks like this can directly be assigned to `this.persistence` now. Before, we had a custom `start()` method and hence I added that indirection. This no longer needs to be typed as IndexedDbPersistence.

Reply:
Done.